contracts/amm/MiltonStorage.sol

```diff
@@ -728,16 +728,18 @@ contract MiltonStorage is
         _balances.treasury = _balances.treasury + incomeFeeValue.toUint128();
 
         if (payoff > 0) {
+            //Buyer earns, Milton looses
+            uint128 finalAbsPayoff = (absPayoff - incomeFeeValue).toUint128();
```

Comment:
this always will be positive?

Reply:
Not always, but for gas optimization we don't check, we assume that income fee percentage never will be higher than 100%